src/main/java/nl/tudelft/planningstool/api/v1/UserOccurrenceAPI.java

```diff
@@ -167,6 +168,18 @@ public UserOccurrenceResponse update(@PathParam("userId") String userId,
         return createListResponse(occurrences);
     }
 
+    @DELETE
+    @Path("/courses/COURSE-{courseId: (\\d|\\w|-)+}/{assignmentId: (\\d)+}")
+    public void deleteOccurrence(@PathParam("userId") String userId, @PathParam("courseId") String courseId, @PathParam("assignmentId") Integer assignmentId) {
+        final Course course = this.courseDAO.getFromUUID(courseId);
+        final User user = this.userDAO.getFromUUID(userId);
+        final Predicate<UserOccurrence> predicate = o -> o.getAssignment().getCourse().equals(course) && o.getAssignment().getId().intValue() == assignmentId.intValue();
```

Comment:
`o.getAssignment().getId().equals(assignmentId)` would be a little less verbose here :wink: